ev-dev-tools/src/ev_cli/templates/interface-Base.hpp.j2

```diff
@@ -120,9 +120,13 @@ private:
                     {% else %}
                         std::vector<{{ arg.array_type }}> {{ arg.name }} = args["{{ arg.name }}"];
                     {% endif %}
+                {% else %}
+                {% if arg.cpp_type == 'Array' %}
+                    auto {{ arg.name }} = {{ var_to_cpp(arg) }}(args["{{ arg.name }}"].get<{{arg.cpp_type}}>());
```

Comment:
Indentation should be fixed, better use a `else if` here.  From looking at the code, I don't get where the difference between `cpp_type == 'Array'` and `'array_type' in arg` is.